tools/crm_resource.c

```diff
@@ -714,6 +725,12 @@ main(int argc, char **argv)
         require_resource = FALSE;
     }
 
+    // --expired without --clear/-U doesn't make sense
+    if (clear_expired == TRUE && rsc_cmd != 'U') {
+        CMD_ERR("--expired requires --clear or -U");
+        argerr++;
+    }
```

Comment:
Thanks for this BTW -- we have very little argument sanity checking in our tools currently :(

Reply:
I'd be happy to add more as I come across it.  There's an awful lot of options, though.  It seems like it'd be kind of endless.